File: cases/taint/python/arithmetic-expression-negative/arithmetic_expression.py
def dfb_source():  # DFB-SOURCE: expression-negative-input
    return 1


def dfb_sink(value):  # DFB-SINK: expression-negative-sink
    pass


def run():
    value = dfb_source()
    computed = (value * 3) + 7  # DFB-WITNESS: expression-negative-computed
    dfb_sink(7)

File: cases/taint/ruby/arithmetic-expression-negative/arithmetic_expression.rb
def dfb_source # DFB-SOURCE: expression-negative-input
  1
end

def dfb_sink(value) # DFB-SINK: expression-negative-sink
end

def run
  value = dfb_source
  computed = (value * 3) + 7 # DFB-WITNESS: expression-negative-computed
  dfb_sink(7)
end
